lib/sportradar/api/nfl/stat/penalties.rb

```diff
@@ -1,29 +1,36 @@
 module Sportradar
   module Api
     class Nfl::Stat::Penalties < Nfl::StatPack
-      def set_stats(data)
-        @response  = data[1] if data.is_a? Array
+      attr_accessor :penalties, :yards
+
+      alias :count :penalties
+
+      def set_stats
+        @response  = response[1] if response.is_a? Array
         @penalties = response['penalties']
         @yards     = response['yards']    
       end
+      def formatted # do we care about using `#display` as the name? it's semantically accurate
```

Comment:
Oops I meant to remove that comment; I originally had it as display and then decided to change it.